pyphare/pyphare/pharein/simulation.py

```
@@ -272,6 +272,204 @@ def check_boundaries(ndim, **kwargs):
+        "reflective",
+        "none",
+        "super-magnetofast-inflow",
+        "super-magnetofast-outflow",
+        "free-pressure-inflow",
+        "fixed-pressure-outflow",
+    )
+    all_directions = ["x", "y", "z"][:ndim]
+    sides = "lower", "upper"
+    boundary_types = kwargs["boundary_types"]
+    physical_directions = []
+    for direction, boundary_type in zip(all_directions, boundary_types):
```

Ruff (0.15.20)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)